# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/tsa/john/locations.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -Nur john-1.6.orig/run/john.ini john-1.6/run/john.ini
--- john-1.6.orig/run/john.ini	1998-12-03 01:29:50.000000000 +0100
+++ john-1.6/run/john.ini	2004-03-11 01:50:33.000000000 +0100
@@ -5,7 +5,7 @@
 
 [Options]
 # Wordlist file name, to be used in batch mode
-Wordfile = ~/password.lst
+Wordfile = /usr/share/john/password.lst
 # Use idle cycles only
 Idle = N
 # Crash recovery file saving delay in seconds
@@ -314,25 +314,25 @@
 
 # Incremental modes
 [Incremental:All]
-File = ~/all.chr
+File = /usr/share/john/all.chr
 MinLen = 0
 MaxLen = 8
 CharCount = 95
 
 [Incremental:Alpha]
-File = ~/alpha.chr
+File = /usr/share/john/alpha.chr
 MinLen = 1
 MaxLen = 8
 CharCount = 26
 
 [Incremental:Digits]
-File = ~/digits.chr
+File = /usr/share/john/digits.chr
 MinLen = 1
 MaxLen = 8
 CharCount = 10
 
 [Incremental:LanMan]
-File = ~/lanman.chr
+File = /usr/share/john/lanman.chr
 MinLen = 0
 MaxLen = 7
 CharCount = 69
diff -Nur john-1.6.orig/src/params.h john-1.6/src/params.h
--- john-1.6.orig/src/params.h	1998-12-03 01:29:50.000000000 +0100
+++ john-1.6/src/params.h	2004-03-11 01:48:33.000000000 +0100
@@ -53,9 +53,9 @@
  * File names.
  */
 #define LOG_NAME			"~/john.pot"
-#define CFG_NAME			"~/john.ini"
+#define CFG_NAME			"/etc/john.ini"
 #define RECOVERY_NAME			"~/restore"
-#define WORDLIST_NAME			"~/password.lst"
+#define WORDLIST_NAME			"/usr/share/john/password.lst"
 
 /*
  * Configuration file section names.
